brianlong/www.validators.app
# 181163219_show_geo_data_in_api

#### What's this PR do?
- Adds latitude and longitude to api response + update the docs

#### How should this be manually tested?
- Use postmant or curl, request validators list or single validator (make sure it has ip record connected) and check if latitude and longitude are present

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/181163219

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
